.eslintrc.js

```diff
@@ -182,7 +182,7 @@ module.exports = {
     "prefer-const": "error",
     "prefer-reflect": "off",
     "prefer-rest-params": "off",
-    "prefer-spread": "error",
+    "prefer-spread": "off",
```

Comment:
p5.play is not transpiled in any way, so no ES6 is supported.  Not sure why this lint option was on, to be honest.